amundsen_application/static/js/components/TableDetail/RequestMetadataForm/index.tsx

```diff
@@ -91,26 +99,29 @@ export class RequestMetadataForm extends React.Component<RequestMetadataProps, R
     const form = document.getElementById("RequestForm") as HTMLFormElement;
     const formData = new FormData(form);
     const recipientString = formData.get('recipients') as string
-    const recipients = recipientString.split(",")
+    const recipients = recipientString.split(RECIPIENT_LIST_DELIMETER)
```

Comment:
I think we should split by `','`. Using the `', '` might introduce bugs if a user adds or deletes extra whitespace.

Reply:
True, let me clean that up.